metaflow/plugins/cards/card_modules/convert_to_native_type.py

```diff
@@ -348,11 +348,11 @@ def _match_partial_type():
         try:
             col_type = str(column_object.dtype)
             if col_type in type_parser:
-                return type_parser[col_type](column_object)
+                return type_parser[col_type](column_object).fillna("null")
```

Comment:
```suggestion
                return type_parser[col_type](column_object.fillna("null"))
```
`type_parser[col_type](column_object)` would return a native type